twill11/hometask-5/4.5.sql

```diff
@@ -0,0 +1,3 @@
+SELECT name, COUNT(film_id) AS number_of_films FROM language 
+LEFT JOIN film ON (language.language_id = film.language_id) 
+GROUP BY language_id
```

Comment:
It doesn't work for me.
`42702] ERROR: column reference "language_id" is ambiguous`